cmd/snowflake/root.go

```diff
@@ -68,7 +67,7 @@ func createChangefeed(cdcServer string, sinkURI *url.URL, tableFQN string, start
 		ReplicaConfig: &cdcv2.ReplicaConfig{
 			Filter: &cdcv2.FilterConfig{Rules: []string{tableFQN}},
 			Sink: &cdcv2.SinkConfig{
-				CSVConfig:          &cdcv2.CSVConfig{IncludeCommitTs: true},
+				CSVConfig:          &cdcv2.CSVConfig{IncludeCommitTs: true, Delimiter: ","},
 				CloudStorageConfig: &cdcv2.CloudStorageConfig{OutputColumnID: putil.AddressOf(true)},
```

Comment:
The default Delimiter in cdcv2 is "", which will lead to [CDC:ErrSinkInvalidConfig]sink config invalid: csv config delimiter cannot be empty